lib/src/arkit_reference_node.dart

```diff
@@ -2,10 +2,12 @@ import 'package:arkit_plugin/src/arkit_node.dart';
 import 'package:arkit_plugin/src/light/arkit_light.dart';
 import 'package:arkit_plugin/src/physics/arkit_physics_body.dart';
 import 'package:vector_math/vector_math_64.dart';
+import 'enums/asset_type.dart';
 
 ///  Node that references an external serialized node graph.
 class ARKitReferenceNode extends ARKitNode {
   ARKitReferenceNode({
+    this.assetType = AssetType.scnassets,
```

Comment:
While these changes might make sense, currently they might rather complicate the usage a bit. You were forced to make `createReferenceNode` return `SCNNode` instead of `SCNReferenceNode`, which sounds wrong to me. Additionally, the `assetType` will be ignored if it is not related to the new functionality, which indicates that it is not related to the current class.

Please create a new class `ARGlbNode` (or something) that will extend `ARKitNode`, revert changes you've done to the `createReferenceNode` parser, and create a new specific `createGlbNode` parser function instead. In the new implementation, please don't use force unwraps too often, instead use `if let` or `guard let` constructions.

Reply:
I create the ARKitGltfNode class and revert min SDK changes.

Thank you for all your comments.